components/mail/thunderbird/thunderbird.p5m

```diff
@@ -32,16 +32,9 @@ license thunderbird.license license="MPL v2.0"
 file files/thunderbird.desktop path=usr/share/applications/thunderbird.desktop
 file usr/lib/$(MACH64)/thunderbird/chrome/icons/default/default48.png path=usr/share/pixmaps/thunderbird-icon.png
 
-link path=usr/bin/thunderbird target=../lib/amd64/thunderbird/thunderbird mediator=thunderbird mediator-version=78
-
```

Comment:
You have removed too much - the link action needs to be kept, it's only the mediator attribute that has to be removed. We still need the link.

Reply:
have the link readded.

Reply:
Version should be 91, right?

Reply:
we remove the mediator, because installation of different versions is not intended